README.md

```diff
@@ -84,37 +84,68 @@ To get images or text on the screen, use the following commands. (Copy and paste
 
 #### `play.new_box()`
 ```python
-box = play.new_box(color='black', x=0, y=0, width=100, height=200, border_color="light blue", border_width=0)
+box = play.new_box(
+    color='black',
+    x=0,
+    y=0,
+    width=100,
+    height=200,
+    border_color="light blue",
+    border_width=10
+    )
 ```
 
 This will put a tall, black box in the middle of the screen.
 
-If you want to change where the image is on the screen, try changing `x=0` (horizontal position) and `y=0` (vertical position). Positive x is to the right and positive y is up. You can also change the color by changing `'black'` to another color name, like `'orange'`.
+If you want to change where the image is on the screen, try changing `x=0` (horizontal position) and `y=0` (vertical position). Just like Scratch, the middle of the screen is (0,0). You can also change the color by changing `'black'` to another color name, like `'orange'`.
 
 
 #### `play.new_image()`
 ```python
-character = play.new_image(image='character.png', x=0, y=0, angle=0, size=100, transparency=100)
+character = play.new_image(
+    image='character.png', 
+    x=0, 
+    y=0, 
+    angle=0, 
+    size=100, 
+    transparency=100
+    )
```

Comment:
minor nitpick, can you double indent (+1 indent) the parameters so its visually  distinct from the parentheses?  e.g.
```
x = something(
    a=1,
    b=2
  )
```

Reply:
ditto for all the other changed examples

Reply:
Done